cocos/ui/UILayoutParameter.h

```diff
@@ -283,12 +285,29 @@ class CC_GUI_DLL LinearLayoutParameter : public LayoutParameter
      */
     LinearGravity getGravity() const;
 
+    /**
+    * Sets size parameter for LayoutParameter.
+    *
+    * @see size
+    * @param size in LinearGravity.
+    */
+    void setLayoutItemSize(Size _size){ _normalSize = _size; }
```

Comment:
The argument shouldn't add _ prefix.  please change `_size` to `size`.
And the comments "@param size in LinearGravity." is also not clear, please fix it.